molecule/gcloud/molecule.yml

```diff
@@ -9,15 +9,15 @@ lint: |
   ansible-lint
 platforms:
   - name: instance
-    image: ubuntu:18.04
+    image: ubuntu:20.04
```

Comment:
Why not 22:04?

Reply:
Baby steps - on the role tests I started by upgrading the EOLD'd Debian 9 to to the current Debian 12 which didn't work and was going to require a ton of changes (to the tests, that is), then to 11 (ditto) and finally to 10 (still supported for a little while) which worked. Ubuntu 22.04 will *probably* work, but I'm being conservative for now.